.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -30,20 +39,32 @@ body:
     validations:
       required: true
 
-  - type: textarea
+  - type: dropdown
     id: environment
     attributes:
-      label: What operating system and browser are you using? (if applicable)
+      label: What operating system, browser and environment are you using? (if applicable)
+      multiple: true
       value: |
-        - Windows 10
-        - Google Chrome 22
+        - Browser - Google Chrome
+        - Browser - Firefox
+        - Browser - Safari
+        - OS - Windows
+        - OS - Mac
+        - OS - Linux
+        - Env - Local
+        - Env - Dev (dgexsol.fr)
     validations:
-      required: false
+      required: true
 
   - type: input
     id: osrdVersion
     attributes:
       label: OSRD version
-      value: "1.0"
+      value: "3dbd5632"
     validations:
       required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        If you have the rights, you can indicate the severity of the bug by adding a label.
```

Comment:
Bugs are usually triaged by contributors, not reporters. I don't think we can ask this.